docs/source/usersguide/scripts.rst

```diff
@@ -0,0 +1,279 @@
+follows the NNDC data convention (1000*Z + A + 300 + 100*m), or the MCNP data
+convention (essentially the same as NNDC, except that the first metastable state
+of Am242 is 95242 and the ground state is 95642).
+
+The optional ``--fission_energy_release`` argument will accept an HDF5 file
+containing a library of fission energy release (ENDF MF=1 MT=458) data. A
+library built from ENDF/B-VII.1 data is released with OpenMC and can be found at
+openmc/data/fission_Q_data_endb71.h5. This data is necessary for
+'fission-q-prompt' and 'fission-q-recoverable' tallies, but is not needed
+otherwise.
+
+-h, --help            show help message and exit
+
+-d DESTINATION, --destination DESTINATION
+                      Directory to create new library in (default: .)
+
+-m META, --metastable META
+                      How to interpret ZAIDs for metastable nuclides. META
+                      can be either 'nndc' or 'mcnp'. (default: nndc)
+
+--xml XML             Old-style cross_sections.xml that lists ACE libraries
+
+--xsdir XSDIR         MCNP xsdir file that lists ACE libraries
+
+--xsdata XSDATA       Serpent xsdata file that lists ACE libraries
+
+--fission_energy_release FISSION_ENERGY_RELEASE
+                      HDF5 file containing fission energy release data
+
+.. _scripts_mcnp70:
+
+------------------------------
+``openmc-convert-mcnp70-data``
+------------------------------
+
+This script converts ENDF/B-VII.0 ACE data from the MCNP5/6 distribution into an
+HDF5 library that can be used by OpenMC. This assumes that you have a directory
+containing files named endf70a, endf70b, ..., endf70k, and endf70sab. The path
+to the directory containing these files should be given as a positional
+argument. The following optional arguments are available:
+
+-d DESTINATION, --destination DESTINATION
+    Directory to create new library in (Default: mcnp_endfb70)
+
+.. _scripts_mcnp71:
+
+------------------------------
+``openmc-convert-mcnp71-data``
+------------------------------
+
+This script converts ENDF/B-VII.1 ACE data from the MCNP6 distribution into an
+HDF5 library that can be used by OpenMC. This assumes that you have a directory
+containing subdirectories 'endf71x' and 'ENDF71SaB'. The path to the directory
+containing these subdirectories should be given as a positional argument. The
+following optional arguments are available:
+
+-d DESTINATION, --destination DESTINATION
+     Directory to create new library in (Default: mcnp_endfb71)
+
+-f FER, --fission_energy_release FER
+     HDF5 file containing fission energy release data
+
+.. _scripts_jeff:
+
+------------------------
+``openmc-get-jeff-data``
+------------------------
+
+This script downloads `JEFF 3.2 ACE data
+<https://www.oecd-nea.org/dbforms/data/eva/evatapes/jeff_32/>`_ from OECD/NEA
+and converts it to a multi-temperature HDF5 library for use with OpenMC. It has
+the following optional arguments:
+
+-b, --batch
+    Suppress standard in
```

Comment:
I think that was originally added by @bhermanmit. I guess the idea is that when you're running a batch job, you wouldn't want live prompts.